lib/logout.js

```diff
@@ -71,7 +71,8 @@ module.exports.logout = function (options) {
           inResponseTo:   data.id,
           destination:    data.serviceProviderLogoutURL || options.destination, // Destination taken from session (matches issuer from the LogoutRequest with session serviceProviderId)
           issuer:         options.issuer, // IdP is the Issuer for this LogoutResponse
-          samlStatusCode: isPartialLogout ? STATUS.PARTIAL_LOGOUT : STATUS.SUCCESS
+          samlStatusCode: isPartialLogout ? STATUS.PARTIAL_LOGOUT : STATUS.SUCCESS,
+          samlStatusMessage: options.samlStatusMessage
```

Comment:
Removing this variable fails when compiling the template. That's why I added it. If empty message won't be set